vmware/container-service-extension
# [VCDA-2477] cluster_script package fix

Signed-off-by: Aniruddha Shamasundar <aniruddha.9794@gmail.com>

To help us process your pull request efficiently, please include: 

Testing done:
* Check if site-packages have cluster_scripts package
* Check if cse can access the cluster_scripts

@rocknes @sahithi @ltimothy7 @sakthisunda


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vmware/container-service-extension/1039)
